.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Continous integration of docker-slim
+
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: ['1.16', '1.17', '1.18']
```

Comment:
it also needs to build with Go 1.15 too (due to a legacy Mac compat requirement)